Issue warning when accessing S3 without specifying anon=True (#482)
dcs4cop/xcube: CHANGES.md

  1. Generator API `xcube.core.gen2.LocalCubeGenerator` and
    `xcube.core.gen2.service.RemoteCubeGenerator`;
  2. Generator CLI `xcube gen2`.
* xcube now issues a warning, if a data cube is opened from object 

>   storage, and credentials have been neither passed not can be found, 

Reviewer: ```suggestion
  storage, and credentials either were passed or can not be found, 
```

Author: not --> nor is correct

Reviewer: I f you want to use ''nor', say:
"credentials neither were passed nor can they be found" .
I don't like this solution as it sounds weird, so I still stand by my suggestion (I just noticed it is missing a "not").

  and the object storage has been opened with the default `anon=False`. (#412)
* xcube no longer internally caches directory listings, which prevents 

>   the situation where a data cube, that has recently been written into object 
>   storage, cannot be found. 

Reviewer: ```suggestion
  the situation where a data cube that has recently been written into object 
  storage cannot be found. 
```
